### PR TITLE
CM-339 Add Faucet Module

### PR DESCRIPTION
**Pull request by njmurarka**
Example use of the faucet:

blzcli tx faucet mintfor $(blzcli keys show neeraj -a) --from vuser --gas-prices 0.01bnt

Has been tested in a single validator environment only.

### CODE REVIEW

**Comment by njmurarka** [comment]
@ebruck Please review and approve ASAP. Need this urgently for a testnet update.

File: Makefile [hunk 1]
```diff
@@ -24,7 +24,7 @@ ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=BluzelleService \
 	-X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT) \
 	-X "github.com/cosmos/cosmos-sdk/version.BuildTags=$(build_tags_comma_sep),cosmos-sdk $(COSMOS_SDK)"
 
-BUILD_FLAGS := -tags "$(build_tags)" -ldflags '$(ldflags)'
+BUILD_FLAGS := -tags "$(build_tags)" -ldflags '$(ldflags)' -tags faucet
```
Comment (ebruck) [inline Makefile:27]: Does this update the version info as well?

**Review by ebruck — CHANGES_REQUESTED** [review]
Please do an interactive rebase to collapse all the diffs. It's hard to tell what is going on.

**Comment by njmurarka** [comment]
> Please do an interactive rebase to collapse all the diffs. It's hard to tell what is going on.

Done.

File: Makefile (hunk 1 above)
Comment (njmurarka) [reply Makefile]: No. Should it?

{"name":"BluzelleService","server_name":"blzd","client_name":"blzcli","version":"0.0.0-40-g62b476f","commit":"62b476f774c9788e8777f5346007a5e05095eebc","build_tags":"ledger,cosmos-sdk v0.38.1","go":"go version go1.13.6 darwin/amd64"}

Comment (njmurarka) [reply Makefile:27]: Now it does.

**Review by ebruck — APPROVED** [review]

**Pull request merged**